packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -292,6 +293,11 @@ export class FluidDataStoreRuntime
 	 */
 	private readonly submitMessagesWithoutEncodingHandles: boolean;
 
+	/**
+	 * @see IFluidDataStoreRuntimeInternalConfig.minVersionForCollab
```

Comment:
I'm not sure if a `@see` block will do anything on its own without some content (the API report says this is undocumented), but also since `FluidDataStoreRuntime` is the thing we expose to consumers as legacy+beta (whereas `IFluidDataStoreRuntimeInternalConfig` is `@internal`) I think the actual docs should be here, and referenced from the `@internal`. Otherwise the pointer is to something that might not even be in the `.d.ts` files?